README.md

```diff
@@ -18,6 +18,8 @@
 
 ```
 yarn install
+cd ios
+pod install
```

Comment:
`brew install cocoapods` might be necessary as well

Reply:
`brew install node` and `npm i -g yarn` I guess